README.md

```diff
@@ -24,7 +24,10 @@ Here are some of the benefits of using this library:
 Feature flags are composed of two parts, a name and a list of feature-filters that are used to turn the feature on.
 
 ### Feature Filters
-Feature filters define a scenario for when a feature should be enabled. When a feature is evaluated for whether it is on or off, its list of feature-filters are traversed until one of the filters decides the feature should be enabled. At this point the feature is considered enabled and traversal through the feature filters stops. If no feature filter indicates that the feature should be enabled, then it will be considered disabled.
+Feature filters define a scenario for when a feature should be enabled. When a feature is evaluated for whether it is on or off, its list of feature-filters are traversed until one of the filters decides the feature should be enabled or disabled. Traversal works in one of two ways:

```

Comment:
Inline with my comment about add-ons in new sections. I wanted to propose keeping the wording the same for this section but just mentioning that it can be overriden with requirement type and linking to the new requirement type section, also moving the new content here into the requirement type section.

Reply:
I feel like we can take one of two approaches-

1. Describe filters in the Any state and then treat RequirementType "All" as an override.
2. Describe filters as having either All or Any logic and describing each option.

When I was writing this- I was using option **2**. Sounds like you're suggesting option **1**. I think this goes along with your other suggestions as well- for moving the RequirementType related info to the RequirementType section. 

I think you're right and we should describe it as an override for now, as it's easier to understand sequentially. I'll re-write it towards that.